api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java

```diff
@@ -449,6 +449,9 @@ private void loadAppsAndExtensions() throws UnsupportedEncodingException {
         if (config.isComponentEnabled(Components.REHAB)) {
             enableRehab();
         }
+        if(config.isComponentEnabled(Components.PERU_PRESCRIPTIONS)){
```

Comment:
```suggestion
        if (config.isComponentEnabled(Components.PERU_PRESCRIPTIONS)) {
```

Reply:
Let's rename this component to "Prescription" and just put a comment "currently only used in Peru" here.